src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+
+const sortList = (array) => {
+  const newList = array.sort(
+    (a, b) =>
+      Number(b.getAttribute('data-salary').replace('$', '').replace(',', '')) -
+      Number(a.getAttribute('data-salary').replace('$', '').replace(',', '')),
+  );
```

Comment:
The `sort` method directly modifies the original array. If you want to keep the original array unmodified, consider using `slice()` to create a shallow copy before sorting.